lib/kube/proxy/forwarder_test.go

```diff
@@ -751,6 +751,7 @@ func TestAuthenticate(t *testing.T) {
 					RouteToCluster:    tt.routeToCluster,
 					KubernetesCluster: tt.kubernetesCluster,
 					ActiveRequests:    tt.activeRequests,
+					Expires:           certExpiration,
```

Comment:
If I add this line, but don't add the fixes above, the test still passes.

This makes me think we don't actually have a test that verifies this behavior is correct.